src/models/EmailLedger.ts

```diff
@@ -1,6 +1,6 @@
 export default interface SCEmailLedger {
-  [domain: string]: {
+  [originalContract: string]: {
+    originalContract: string
```

Comment:
Why do we need to map `originalContract` to `originalContract` if we can use the `key`?